testing/README.md

```diff
@@ -10,13 +10,18 @@ Checkout `ivadomed/.github/workflows/run_tests.yml` to see how tests are run on
 
 ## Running Locally
 
-1. Download the required dataset(s) using the `ivadomed` command line tools:
+1. Install dependencies
+```
+pip install pytest pytest_console_scripts
```

Comment:
I agree these are the minimal essential. There MAY be more dev related dependencies in #1129 here you want to take a look and see: https://github.com/ivadomed/ivadomed/pull/1129/files#diff-c9c796259f3852b51b531b79cbf07820145088d4f108fb006ab44b15f0b15934 . For now, pytest-cases are not used but I suspect it will be in the near future.

Reply:
#1129/#1000 put `pytest` into `[dev]`, and that's what CI uses:

https://github.com/ivadomed/ivadomed/blob/14f67f99812599840ef2195171cffc17feb3f643/.github/workflows/run_tests.yml#L66

So I think we should recommend that instead:

```suggestion
cd ivadomed  # root of the repo
pip install -e .[dev]
```